### PR TITLE
Create tests for ProxyNetworkConfigDTO

### PR DESCRIPTION
**Pull request by antonRMNV**
Tests for ProxyNetworkConfigDTO were created.

### CODE REVIEW

File: src/test/java/executor/service/model/ProxyNetworkConfigDTOTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+class ProxyNetworkConfigDTOTest {
+
+    ProxyNetworkConfigDTO proxyNetworkConfig1;
+    ProxyNetworkConfigDTO proxyNetworkConfig2;
+    ProxyNetworkConfigDTO proxyNetworkConfig3;
+
+    @BeforeEach
+    void setUpEach() {
+        proxyNetworkConfig1 = new ProxyNetworkConfigDTO("localhost", 8080);
```
Comment (Serhx4) [inline src/test/java/executor/service/model/ProxyNetworkConfigDTOTest.java:15]: @antonRMNV, we can improve our code by using constants for "magic" numbers and strings: `localhost`, `somehost`, `8080`, `80`.
These numbers and strings are used in almost every test, so good practice is to use constant fields for them.

**Review by Serhx4 — CHANGES_REQUESTED** [review]

File: src/test/java/executor/service/model/ProxyNetworkConfigDTOTest.java (hunk 1 above)
Comment (antonRMNV) [reply src/test/java/executor/service/model/ProxyNetworkConfigDTOTest.java:15]: @Serhx4, ok, i'll improve this code by adding constants. Thanks.

**Review by AlexandrTokarenko — APPROVED** [review]

File: src/test/java/executor/service/model/ProxyNetworkConfigDTOTest.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
+package executor.service.model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class ProxyNetworkConfigDTOTest {
+
+    private static final String LOCALHOST = "localhost";
```
Comment (artemkedyk) [inline src/test/java/executor/service/model/ProxyNetworkConfigDTOTest.java:9]:
```suggestion
    private final String LOCALHOST = "localhost";
```

Comment (antonRMNV) [reply src/test/java/executor/service/model/ProxyNetworkConfigDTOTest.java]: @artemkedyk, corrected. Thanks.

**Review by Serhx4 — APPROVED** [review]

**Review by tatarynovdima — APPROVED** [review]

**Review by bielik-b — APPROVED** [review]

**Review by artemkedyk — APPROVED** [review]

**Pull request merged**